CODEOWNERS

```
@@ -21,3 +21,4 @@
 # CI
 /.github/ @paritytech/ci @fevo1971
 /.gitlab-ci.yml @paritytech/ci @fevo1971
+/*.Dockerfile @paritytech/ci
```

Will this catch a single `Dockerfile` if it's ever introduced to the repo?

```suggestion
/*Dockerfile @paritytech/ci
```